models/yang/sonic/sonic-loopback-interface.yang

```diff
@@ -0,0 +1,54 @@
+module sonic-loopback-interface {
+	namespace "http://github.com/Azure/sonic-loopback-interface";
+	prefix sonic-lo-if;
+
+	import ietf-inet-types {
+		prefix inet;
+	}
+
+	import sonic-port {
+		prefix prt;
+	}
+
+	organization
+		"SONiC";
+
+	contact
+		"SONiC";
+
+	description
+		"SONIC LOOPBACK INTERFACE";
+
+	revision 2019-07-02 {
+		description
+			"Initial revision.";
+	}
+
+	container sonic-loopback-interface {
+
+		container LOOPBACK_INTERFACE {
+
+			list LOOPBACK_INTERFACE_LIST {
+				key "loIfName";
+
+				leaf loIfName{
+                    type string;
```

Comment:
Any pattern for the loopback name ?

Reply:
Could you tell me what is the maximum number of supported loopback interfaces? I searched for it, couldn't find it.

Reply:
@dutta-partha - We can update this when we bring in loopback interface changes.

Reply:
@bhavini-gada How are we going to track this review comment for addressing later ?

Reply:
@bhavini-gada @justinejose91 Please check the review comments in PR #18 for sonic-acl.yang etc., we should make sure to add constraints so that avoid similar review comments in future PR.